testsuite/unittests/test_gap_junction.sli

```diff
@@ -82,7 +98,7 @@ neuron_no_gap neuron_no_gap Connect
 0 /max_delay get /old_max Set
 0 /min_delay get /old_min Set
 
-neuron_gap neuron_gap 4.0 4.0 /gap_junction Connect
+neuron_no_gap neuron_no_gap 2.0 2.0 Connect
```

Comment:
I agree with @heplesser on this part. we can not use gap junctions here as we explicitly check before that setting a delay is not possible for these. it seems to me like this code part is obsolete.